client/state/action-types.js

```diff
@@ -137,6 +137,7 @@ export const CONNECTED_APPLICATIONS_RECEIVE = 'CONNECTED_APPLICATIONS_RECEIVE';
 export const CONNECTED_APPLICATIONS_REQUEST = 'CONNECTED_APPLICATIONS_REQUEST';
 export const CONNECTION_LOST = 'CONNECTION_LOST';
 export const CONNECTION_RESTORED = 'CONNECTION_RESTORED';
+export const CONNECTIONS_EXPANDED_SERVICE = 'CONNECTIONS_EXPANDED_SERVICE';
```

Comment:
This may be just a personal preferences, but I prefer actions to be verbs, as `CONNECTIONS_EXPANDED_SERVICE` doesn't indicate what the action is doing clearly, where `CONNECTIONS_SET_EXPANDED_SERVICE` or `CONNECTIONS_EXPANDED_SERVICE_SET` does a little better, but there is a range of approaches in this file already, so not critical to getting this over the line 😃